sources/sections-en/04-non-si-units.adoc

```diff
@@ -38,7 +38,9 @@ Nonetheless it is recognized that some non-SI units are widely used and are expe
 ====
 The gal (symbol: stem:["Gal"]) is a non SI unit of acceleration employed in geodesy and geophysics to express acceleration due to gravity
 
-stem:[1 " Gal" = 1 " cm s"^(−2) = 10^(−2) " m s"^(−2)]
+[align=left]
```

Comment:
Align left needed? Feels a bit strange...?

Reply:
Apparently, it is needed for Word output according to the [documentation](https://www.metanorma.com/author/topics/document-format/text/), that says:

*If you have line-breaks in a paragraph, and the default alignment in the stylesheet is justified (as is often the case in Word output), you will need to set `[align=left]` to make the paragraph look as normally expected*

Reply:
I see! This document is only generated via PDF, so perhaps we don't need it here. Thanks.

Reply:
But maybe in this case it doesn't apply? @opoudjis?

Reply:
> I see! This document is only generated via PDF, so perhaps we don't need it here. Thanks.

Oh, I hadn't read this. Ok, changes applied!